### PR TITLE
Modeling Rule Testing Test Data File Examples

### PR DESCRIPTION
**Pull request by avidan-H**
## Status
- [x] Ready

## Related Issues
Relates to: https://github.com/demisto/demisto-sdk/pull/2482

## Description
- Adds a `*_testdata.json` file for the `Jamf` pack's modeling rule (version 1.3) which is based on JSON event logs
- Adds a `*_testdata.json` file for the `ApacheTomcat` pack's modeling rule (version 1.3) which is based on Syslog event logs

## Does it break backward compatibility?
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by avidan-H** [comment]
I am adding the `docs-approved` label since there is no documentation that needs to be reviewed.

File: Packs/ApacheTomcat/ModelingRules/ApacheTomcatModelingRules_1_3/ApacheTomcatModelingRules_1_3_testdata.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+    "data": [
+        {
+            "test_data_event_id": "b71645d8-853e-498f-8545-f60f3085cbea",
```
Comment (tkatzir) [inline Packs/ApacheTomcat/ModelingRules/ApacheTomcatModelingRules_1_3/ApacheTomcatModelingRules_1_3_testdata.json:4]: Can this (and all other id's) be changed to a value that's clearly bogus?

**Review by tkatzir — APPROVED** [review]
Approved, with a comnnent.

File: Packs/ApacheTomcat/ModelingRules/ApacheTomcatModelingRules_1_3/ApacheTomcatModelingRules_1_3_testdata.json (hunk 1 above)
Comment (avidan-H) [reply Packs/ApacheTomcat/ModelingRules/ApacheTomcatModelingRules_1_3/ApacheTomcatModelingRules_1_3_testdata.json:4]: I believe only if it's a valid UUID - so any sequence where there are 8-4-4-4-12 alphanumeric characters.

**Pull request merged**